docs/Advanced-Usage/Conda.rst

```diff
@@ -0,0 +1,183 @@
+Non-Source Dependency Management
+================================
+
+In :doc:`/Initial-Setup/Setting-up-your-Manager-Instance`, we quickly copy-pasted the contents
+of ``scripts/machine-launch-script.sh`` into the EC2 Management Console and
+that script installed many dependencies that FireSim needs using
+`conda <https://conda.io/en/latest/index.html>`_,  a platform-agnostic package
+manager, specifically using packages from the `conda-forge community <https://conda-forge.org/#about>`_.
+
+In many situations, you may not need to know anything about conda.  By default, the
+``machine-launch-script.sh`` installs conda and all of the FireSim dependencies into ``/opt/conda``
+and adds the required setup to the system-wide ``/etc/profile.d/conda.sh`` init script to add
+``/opt/conda`` to everyone's path.
+
+However, the script is also flexible.  For example, if you do not have root access, you can specify
+an alternate install location with the ``--prefix`` option to ``machine-launch-script.sh``.  The only requirement
+is that you are able to write into the install location.  See ``machine-launch-script.sh --help`` for more details.
+
+.. warning::
+
+    Without root access you can do many things, like :doc:`/Building-a-FireSim-AFI`,
+    :ref:`meta-simulation` of a FireSim system using Verilator or even developing new features in FireSim.
+    However, to :ref:`run a simulation on a F1 FPGA <running_simulations>` , FireSim currently requires that
+    you are able to act as root via ``sudo``.
+
+Updating a Package Version
```

Comment:
Need to add a section near the top of the page that explains some of the quirks with using the compiler toolchain.

You may have come to this page saying, "Where the heck is gcc?".  Well, welcome to using the conda toolchain.  Explain that activating the compilers sets the standard gnumake environment variables to direct make to the correct toolchain but if you are building something that hardcodes `gcc` instead of using `$CC`, you are going to notice it.  

Essentially give an updated version of https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html that points to the conda-forge toolchain information.

ALSO: it is possible for us to create a package that installs the short-names for the tools into /bin as symlinks to the longer tool names.  However, I dislike doing that because it's extra work and build flows shouldn't suck and hardcode toolnames in them.

Reply:
I actually switched to doing this.  The conda-forge guys actually started making a package that is just the symlinks of the shortnames to the longnames and I'm adding the minimal set of options you have to have to make it act like a system-packaged toolchain in a PR that should be merged soon.  FYI, it's the specfile hacking that I currently have at the end of machine-launch-script.sh

Reply:
b3d7ff69ee6c4be23e1b23570c9b216c8af5bec4 is the commit where I switched to using the package that installs the shortnames instead of the one that uses the activation scripts with all of the default options for building packages shoved into FLAGS env vars.

Reply:
Didn't need to explain a bunch of details about the compilers because we switched away from using the activation scripts and it is more straightforward.